src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -8,16 +11,18 @@ import { ViewingService } from '../../../core/viewing.service';
 })
 
 export class MessagesComponent implements OnInit {
-  userMsgData;
-
-  constructor(public viewingService: ViewingService) { }
+  fonts: Fonts;
+  constructor(public viewService: ViewingService, public authService: AuthService, public fontService: FontService) { }
```

Comment:
Since everything in fontService is static, you should remove it from here.
```suggestion
  constructor(public viewService: ViewingService, public authService: AuthService) { }
```